tools/packaging/static-build/ovmf/build-ovmf.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+#
+# Copyright (c) 2022 IBM
+#
+# SPDX-License-Identifier: Apache-2.0
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+source "${script_dir}/../../scripts/lib.sh"
+
+ovmf_repo="${ovmf_repo:-}"
+ovmf_dir="edk2"
+ovmf_version="${ovmf_version:-}"
+kata_version="${kata_version:-}"
+
+set +u
+ovmf_build="${ovmf_build:-sev}"
+architecture="X64"
+toolchain="GCC5"
+build_target="RELEASE"
+
+if [ -z "$ovmf_repo" ]; then
+       info "Get ovmf information from runtime versions.yaml"
+       ovmf_url=$(get_from_kata_deps "externals.ovmf.url" "${kata_version}")
```

Comment:
`git clone https://github.com/tianocore/edk2` is enough to clone the repo, you don't need to append `.git` in the url. So this block can be simplified.